packages/api/scripts/init.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+if [ $1 == 'testing' ]
+then
+  echo "--- Seeding DB with test data ---"
+  for i in `seq 1 10`;
+    do
+      nc -z litefarm-db 5433 && echo Success && break
+      echo -n .
+      sleep 1
+    done
+  # echo Failed waiting for Postgres && exit 1
```

Comment:
Perhaps is good to remove this commented line, I left it there as a personal note and forgot to delete it. Up to you of course.